### PR TITLE
disable concurrent build for Jenkins

### PR DESCRIPTION
**Pull request by athiruma**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -58,6 +58,8 @@
         'google-api-python-client==2.57.0',  # google drive
         'google-auth-httplib2==0.1.0',  # google drive
         'google-auth-oauthlib==0.5.2',  # google drive
+        # ldapsearch installation for rhel/centos - python3.9
+        # sudo dnf install -y python39-devel openldap-devel gcc
         'python-ldap==3.4.2',  # ldapsearch
```
Comment (ebattat) [inline setup.py:63]: 'python-ldap==3.4.2',  # manual installation: sudo dnf install -y python39-devel openldap-devel gcc

File: jenkins/tagging/Jenkinsfile [hunk 2]
```diff
@@ -1,4 +1,7 @@
 pipeline {
+    options {
+        disableConcurrentBuilds()
+    }
```
Comment (ebattat) [inline jenkins/tagging/Jenkinsfile:4]: Did you test it, if it work lets enable it in all the jobs

Comment (athiruma) [reply jenkins/tagging/Jenkinsfile:4]: yes, worked.

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**